app/src/main/res/values-de/strings.xml

```diff
@@ -42,7 +42,7 @@
     <!-- Browser menu button that opens the settings menu -->
     <string name="browser_menu_settings">Einstellungen</string>
     <!-- Browser menu button that opens a user's library -->
-    <string name="browser_menu_your_library">Ihre Bibliothek</string>
+    <string name="browser_menu_your_library">Bibliothek</string>
```

Comment:
Why was the "Your" from" Your Library" removed in the German translation? It was an explicit decision a few months ago that this menu item should be "Your Library" and not only "Library". And it's still "Your Library" in the original string.

Reply:
CC @Pike @Delphine

Reply:
You can see on https://pontoon.mozilla.org/de/android-l10n/mozilla-mobile/fenix/app/src/main/res/values/strings.xml/?search=browser_menu_your_library&string=198474 that this change was made by @Archaeopteryx

Reply:
Whose else library shall it be? It's also not addressed as "Your library" but as 'Library' in the view it opens and similar concepts like 'Collections' don't use 'Your'. Avoiding 'Your' and similar words caters to both audiences preferring to be addressed formally and casually.

Reply:
Instead of just changing a string only for one language should be discussed with UX so that every language does the same.

Please also note that it was already discussed in the past. In fact I wrote a PR to change "Your Library" to "Library" a few months ago [1] but it was reverted because UX wanted the "Your Library" [2] string, so it was an explicit choice *not* to use "Library". There is also an explanation why UX made this decision.

Also note that Firefox for iOS also uses "Your Library" and both products should use the same wording. Before my string change was reverted I also asked the Firefox for iOS team to change the string for the sake of consistency but they don't want to change "Your Library" to "Library" either [3].

[1] https://github.com/mozilla-mobile/fenix/pull/1440
[2] https://github.com/mozilla-mobile/fenix/issues/469#issuecomment-485878998
[3] https://github.com/mozilla-mobile/firefox-ios/issues/4814

Reply:
@Pike Can you change it back? I can't suggest the change on Pontoon because it's identical to an older translation.

Reply:
I've talked with Michelle from UX, and it is acceptable for a localizer to drop a pronoun and adapt tone depending on product and country. However it's important to remain consistent throughout products.
@mheubusch can elaborate more on this if needed.